### PR TITLE
Bug/1700 manifest autload

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Addresses issue #1700

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Review by aaemnnosttv — APPROVED** [review]
LGTM 👍

**Comment by aaemnnosttv** [comment]
The build is failing but it's unrelated to these changes it seems, but rather the new version of the AMP plugin that we install for E2E tests.

**Comment by eugene-manuilov** [comment]
@aaemnnosttv so what's the plan? Do we need to wait till e2e are fixed or we can go ahead and merge this PR to unblock smoke testing?

**Comment by aaemnnosttv** [comment]
@eugene-manuilov it can't be merged as long as checks are failing – GitHub won't allow it.

We can use the zip built from the action though to test with.

File: .github/workflows/zips.yml
Comment (felixarntz) [inline .github/workflows/zips.yml:34]: Why did this need to be updated?

Comment (eugene-manuilov) [reply .github/workflows/zips.yml:34]: @felixarntz because `php-actions/composer` generates files owned by `root` user. It causes permission denied errors when we try to re-generate autoload map.

Comment (aaemnnosttv) [reply .github/workflows/zips.yml:34]: @felixarntz I believe there was a permissions issue due to ownership differences between composer which runs in the action's container and the "local" composer since we now run composer during the zip command which uses the "local" composer binary and user permissions.

**Pull request merged**
